stdlib/stdlib.mli

```diff
@@ -1065,6 +1065,30 @@ module LargeFile :
   regular integers (type [int]), these alternate functions allow
   operating on files whose sizes are greater than [max_int]. *)
 
+(** {1 Option type} *)
+
+type nonrec 'a option = 'a option = None | Some of 'a
+(** The type of optional values of type ['a]. *)
+
+(** {1 Int type} *)
+
+type nonrec int = int
+(** The type of integer numbers. *)
+
+(** {1 Bool type} *)
+
+type nonrec bool = bool = false | true
+(** The type of booleans (truth values). *)
+
+(** {1 Unit type} *)
+
+type nonrec unit = unit = ()
+(** The type of the unit value. *)
+
+(** {1 Exceptions} *)
+
+type nonrec exn = exn
```

Comment:
Shouldn't this be: `type nonrec exn = exn = ..`? As it stands, I think your change breaks this program:

```ocaml
module M : sig type t = .. end = struct type t = exn = .. end;;
```

Reply:
Yes; fixed.